src/github.com/matrix-org/go-neb/database/schema.go

```diff
@@ -262,6 +262,35 @@ func selectRealmTxn(txn *sql.Tx, realmID string) (types.AuthRealm, error) {
 	return realm, nil
 }
 
+const selectRealmsByTypeSQL = `
+SELECT realm_id, realm_json FROM auth_realms WHERE realm_type = $1
```

Comment:
I think the behaviour of ``projectToRealm`` depends on the order that the realms are returned in?